lib/api_watch_diff.ml

```diff
@@ -38,9 +70,12 @@ let diff_value ~typing_env ~val_name ~reference ~current =
   | exception Includecore.Dont_match _ -> Some ()
 
 let compare_values ~reference ~current =
-  let env = env_setup ~ref_sig:reference ~curr_sig:current in
+  let modified_current =
+    add_manifest_to_abstract_types ~ref_sig:reference ~curr_sig:current
+  in
```

Comment:
This should probably be moved to the `env_setup` function since this is done specifically to setup the env properly and will be useful not only to compare values but any other item.

We don't want to compare the modified items but the original ones here.